data_structures/binary_tree/is_sorted.py

```diff
@@ -0,0 +1,96 @@
+"""
+Given the root of a binary tree, determine if it is a valid binary search tree (BST).
+
+A valid binary search tree is defined as follows:
+- The left subtree of a node contains only nodes with keys less than the node's key.
+- The right subtree of a node contains only nodes with keys greater than the node's key.
+- Both the left and right subtrees must also be binary search trees.
+
```

Comment:
```suggestion

In effect, a binary tree is a valid BST if its nodes are sorted in ascending order.
```
Added a note to explain how "is valid BST" relates to "is sorted"